backend/ecs_tasks/delete_files/main.py

```diff
@@ -104,26 +118,30 @@ def execute(queue_url, message_body, receipt_handle):
             logger.info("Generating new parquet file without matches")
             out_sink, stats = delete_matches_from_file(infile, cols)
         if stats["DeletedRows"] == 0:
-            raise ValueError(
-                "The object {} was processed successfully but no rows required deletion".format(
-                    object_path
+            if all_files:
```

Comment:
Looks like you are adding a setting to skip throwing for no deletions in file, can I know the reason for that?